Examples/Io/Root/include/ActsExamples/Io/Root/RootTrackStatesWriter.hpp

```diff
@@ -145,90 +147,90 @@ class RootTrackStatesWriter final : public WriterT<ConstTrackContainer> {
   std::vector<float> m_pull_y_hit;  ///< hit pull y
   std::vector<int> m_dim_hit;       ///< dimension of measurement
 
-  std::array<int, 4>
+  std::array<int, eSize>
       m_nParams{};  ///< number of states which have
                     ///< filtered/predicted/smoothed/unbiased parameters
```

Comment:
maybe we can update this commentary style? it doesn't match our usual documentation style.